### PR TITLE
predictions and models

### PR DESCRIPTION
**Pull request by ataymano**
some helpers for picking up outputs (predictions/models) produced by vw

### CODE REVIEW

File: from_mwt_ds/DataScience/vw_executor/artifacts.py [hunk 1]
```diff
@@ -101,3 +106,58 @@ def metrics(self):
         if not self._processed:
             self._process()
         return self._metrics
+
+class Predictions(Artifact):
+    def __init__(self, path):
+        super().__init__(path)
+
+    @property
+    def cb(self):
```
Comment (olgavrou) [inline from_mwt_ds/DataScience/vw_executor/artifacts.py:115]: add one for cats maybe? preds look like `action:pdf_value`, so similar to `cb` but with only one prediction per line. Now that I think about it, is there a corner case for cb where we only have one action?

Comment (ataymano) [reply from_mwt_ds/DataScience/vw_executor/artifacts.py:115]: added cats one
added corner cases to tests

**Review by olgavrou — APPROVED** [review]
